go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/NodeFactoryIo/vedran
+
+go 1.15
+
+require (
+	github.com/eugenmayer/go-exec v1.0.0
+	github.com/getsentry/sentry-go v0.7.0
+	github.com/magiconair/properties v1.8.1
+	github.com/spf13/cobra v1.0.0
+	github.com/spf13/viper v1.7.1
```

Comment:
I don't think you need both github.com/magiconair/properties and github.com/spf13/viper . We can support loading cli config from file but afaik not in current scope